tests/Behat/DrupalContext.php

```diff
@@ -126,4 +127,30 @@ public static function disableDatetimeTesting(AfterFeatureScope $scope): void {
     \Drupal::service('module_installer')->uninstall(['datetime_testing']);
   }
 
+  /**
+   * Creates a non-EU language for the given scenario.
+   *
+   * @param \Behat\Behat\Hook\Scope\BeforeScenarioScope $scope
+   *   The scope.
+   *
+   * @BeforeScenario @enable-non-eu-language
```

Comment:
Why we don't have a parameter to pass which language to create here? We might use this later. I just noticed. Perhaps a step like this already exists?

Reply:
Ah nevermind, i thought it was a step.